docs/config/index.md

```diff
@@ -81,14 +81,14 @@ All configuration options that are not supported inside a [workspace](/guide/wor
 ### include
 
 - **Type:** `string[]`
-- **Default:** `['**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}']`
+- **Default:** `['**/__tests__/**/*.?([cm])[jt]s?(x)', '**/?(*.){test,spec}.?([cm])[jt]s?(x)']`
```

Comment:
The `?([cm])` parts could be `?(c|m)`. Same applied to other patterns as well.

```suggestion
- **Default:** `['**/__tests__/**/*.?(c|m)[jt]s?(x)', '**/?(*.){test,spec}.?(c|m)[jt]s?(x)']`
```

Reply:
I was picking visual consistency over brevity, though I'm not against this change